tests/std/include/range_algorithm_support.hpp

```diff
@@ -87,7 +87,7 @@ namespace test {
 
     enum class CanDifference : bool { no, yes };
     enum class CanCompare : bool { no, yes };
-    enum class ProxyRef : bool { no, yes };
+    enum class ProxyRef { no, yes, prvalue };

```

Comment:
This really is an abominable hack. I've added a item in the post-merge cleanup task list (https://github.com/microsoft/STL/issues/39#issuecomment-665333818) to change this to a general "style of reference" enumeration instead of the previous `yes,no` or the current [`true,false,file_not_found`](https://thedailywtf.com/articles/What_Is_Truth_0x3f_).